test/evergreen.yml

```diff
@@ -1000,7 +1000,9 @@ tasks:
           DGNU_CXX_VERSION: -DDGNU_CXX_VERSION=9
 
   - name: compile-clang
-    tags: ["pull_request", "pull_request_compilers"]
+    # The task is disabled as a package is missed on the ubuntu2004-wt-large distro.
+    # We can re-enable the task once the missing package is added. WT-10595.
```

Comment:
FIXME?

Reply:
FIXME added in [aad128](https://github.com/wiredtiger/wiredtiger/pull/8801/commits/7aad128fd28cd226d710051f1a027f90813d5258)